R/fortify.anosim.R

```diff
@@ -0,0 +1,49 @@
+##' @title Fortify analysis of similarities (ANOSIM) results
+##' @description Prepares fortified versions of results from \code{\link[vegan]{anosim}} objects.
+##' @details TODO
+##' @param model an object of class \code{\link[vegan]{anosim}}.
+##' @param data original data set. Currently ignored.
+##' @param what character; what aspect of the results should be fortified? \code{"ranks"} returns ranks of dissimilarity entries and names of classies. \code{"stats"} returns significance from permutation  and the value of ANOSIM statistic R.
```

Comment:
Trivial ypo: double space between `permutation` and `and`

Reply:
Corrected.